javav2/example_code/dynamodb/src/main/java/com/example/dynamodb/ScenarioPartiQ.java

```diff
@@ -0,0 +1,343 @@
+//snippet-sourcedescription:[ScenarioPartiQ.java demonstrates how to perform various Amazon DynamoDB operations using PartiQL.]
+//snippet-keyword:[SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-service:[Amazon DynamoDB]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[02/11/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.dynamodb;
+
+// snippet-start:[dynamodb.java2.scenario.partiql.import]
+import com.fasterxml.jackson.databind.JsonNode;
+import software.amazon.awssdk.core.waiters.WaiterResponse;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+import software.amazon.awssdk.enhanced.dynamodb.Key;
+import software.amazon.awssdk.enhanced.dynamodb.model.QueryConditional;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+import software.amazon.awssdk.services.dynamodb.waiters.DynamoDbWaiter;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+// snippet-end:[dynamodb.java2.scenario.partiql.import]
+
+
+/*
+ *  
+ *  Before running this Java code example, set up your development environment, including your credentials.
+ *
+ *  For more information, see the following documentation topic:
+ *
+ *  https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ *
+ *  This Java example performs these tasks:
+ *
+ * 1. Create the Amazon DynamoDB Movie table with partition and sort key.
+ * 2. Put data into the Amazon DynamoDB table from a JSON document.
+ * 3. Add a new item.
+ * 4. Get an item by the composite key (the Partition key and Sort key).
```

Comment:
4. Gets an item by the composite key (the partition key and sort key).

Reply:
fixed